packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -170,6 +173,47 @@ if (compatKind !== undefined) {
     configList = configList.filter((value) => compatKind!.includes(value.kind));
 }
 
+function getUnexpectedLogErrorException(logger: ErrorTrackingLogger){
+    const results = logger.results();
+    if(results.unexpectedErrors.length > 0){
+        return new Error(
+            // eslint-disable-next-line max-len
+            `Unexpected Errors in Logs. Use itExpects to specify expected errors:\n${ JSON.stringify(results.unexpectedErrors, undefined, 2)}`);
```

Comment:
nit:
`Unexpected Errors in Logs. Use itExpects to specify expected errors:\n` +
`${ JSON.stringify(results.unexpectedErrors, undefined, 2)}`
instead of disable max len

Reply:
@markfields  to you above question. this is the error throw if unexpected events are hit. it tell you to use itexpect, and give you the full list of unexpected events, or the unhit expected event in the next block